articles/setting-up-cicd-for-python-packages-using-github-actions/index.md

```diff
@@ -0,0 +1,279 @@
+# Setting up CI/CD for Python Packages using GitHub Actions
+
+## Outline
+
+- Prerequisites
+- What is CI/CD
+- Building a Python Package
+- Authenticating GitHub with Test PyPI
+- Packaging & Deploying with GitHub Actions
+- Testing Python Package Locally
+- Conclusion
+
+## Prerequisites
+
+- Understanding of git and GitHub
+- Creating a GitHub repository
+- Creating a Test PyPI account
+- Understanding how to [Build Python Packages](https://www.section.io/engineering-education/building-a-python-package-and-publishing-on-pypi/)
```

Comment:
```suggestion
- Understanding of Git and GitHub.
- Creating a GitHub repository.
- Creating a test PyPI account.
- Understanding how to [Build Python Packages](/engineering-education/building-a-python-package-and-publishing-on-pypi/).
```
If you are referencing another section article, you need not specify the entire URL. Instead, use `/engineering-education/building-a-python-package-and-publishing-on-pypi/`. It minimizes the time for webpage reload.

Reply:
Hi @srishilesh for the **Test PyPI...**, Test is actually a pronoun as it's a separate PyPI used for tests. That's why I use the capital letter T and not a small letter. We can find more information [here](https://test.pypi.org/)

Reply:
Thank you for letting me know.

Please modify the changes accordingly.